docs/niscope/attributes.rst

```diff
@@ -2144,13 +2144,11 @@ niscope.Session properties
 
 .. py:attribute:: meas_array_gain
 
-    Every element of an array is multiplied by this scalar value during the Array Gain measurement.  Refer to :py:data:`~niscope.NISCOPE_VAL_ARRAY_GAIN` for more information.
+    Every element of an array is multiplied by this scalar value during the Array Gain measurement.  Refer to :py:data:`~niscope._ArrayMeasurement.ARRAY_GAIN` for more information.
```

Comment:
Are public attributes using a private enum? Is this a mistake in our metadata?

Reply:
If you look, this is not the enum. This is just documentation. This attribute does not have an associated enum. If you look at what is changing, it's just now that we are adding the referenced enum back in, it will link to that instead of not being a valid link.

However, given what this attribute is for, I think the attribute should be private as part of making the measurement library private.